src/Tests/CustomCacheTagsTest.php

```diff
@@ -32,6 +32,8 @@ class CustomCacheTagsTest extends WebTestBase {
   public static $modules = array(
     'node',
     'views',
+    'menu_ui',
```

Comment:
Hm. I actually don't care about those modules in the test, enabling the makes it slower. Instead, let's remove them from the configuration, menu_ui is in the node.type files, remove the dependency and the additional settings, path is in the form display files, remove the path filed and module dependency there.